src/elements/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import './Button.css';
+import clsx from 'clsx';
+
+type ButtonProps = {
+    text: string;
+    className?: string | string[] | { [key: string]: boolean };
+}
+
+function Button({ text, className }: ButtonProps) {
+    return (
+        <button className={clsx(className)}>{text}</button>
```

Comment:
I don't think we should allow classes to be passed via props. That allows people to have control over the component they probably shouldn't have.

What are your thoughts on variation and descriptor props to handle how the button changes:

Example:
```jsx
<Button
    variation="outlined" // or "contained" or "text"
    size="large" // or "small" or "medium"
    disabled={false}
    loading={true}
/>
```

And based on those props, you use clsx you combine different classes in this file from the Button.css